### ISSUE TITLE
Social Feed for Promotional Pages

### ISSUE THREAD

**Issue by cckath** [massgov/feature-requests#213]
We would like to see a social feed added to promotional and agency pages to better feature content across channels. Thank you!

**Reference**
Include JIRA ticket number or ServiceNow ticket numbers that correspond to this issue.

**Comment by joeg8612** [comment]
Hi, just to clarify, would you be interested in embedding a social feed on the page?  For example, you see recent tweets?  Can you give an example of a page where you would like to do that?

**Comment by cckath** [comment]
Yes, thank you. I would like to embed a social feed on a promotional page that we plan to launch this summer. However, I would also like the ability to include a social feed on our existing agency website at www.mass.gov/mcb<http://www.mass.gov/mcb>. Either Twitter or Facebook would be the preferred channels for the feed. An example promotional page that we are going to mimic for style and layout can be found here:
https://www.mass.gov/technology-careers-at-the-commonwealth-of-massachusetts

Please let me know if you need anything further, and thank you for looking into this for us!

Best,

Carla Kath
Director of Communications
Massachusetts Commission for the Blind
600 Washington Street
Boston, MA 02111
Office: (617) 626-7586
Mobile: (857) 301-5188
Fax: (617) 422-0419
Carla.Kath@mass.gov

MCB Confidentiality Notice: This e-mail may contain confidential information that may also be legally privileged and that is intended only for the use of the Addressee(s) named above.  If you are not the intended recipient or an authorized agent of the recipient, please be advised that any dissemination or copying of this e-mail, or taking of any action in reliance on the information contained herein, is strictly prohibited.  If you have received this e-mail in error, please notify me immediately and then delete the email from your system.
